closed/OpenJ9.gmk

```diff
@@ -462,13 +469,19 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
   else # OPENJ9_ENABLE_INLINE_TYPES
     CMAKE_ARGS += -DJ9VM_OPT_VALHALLA_VALUE_TYPES=OFF
   endif # OPENJ9_ENABLE_INLINE_TYPES
-  
+
   ifeq (true,$(OPENJ9_ENABLE_CRAC_SUPPORT))
     CMAKE_ARGS += -DJ9VM_OPT_CRAC_SUPPORT=ON
   else # OPENJ9_ENABLE_CRAC_SUPPORT
     CMAKE_ARGS += -DJ9VM_OPT_CRAC_SUPPORT=OFF
   endif # OPENJ9_ENABLE_CRAC_SUPPORT
 
+  ifeq (true,$(OPENJ9_ENABLE_JFR))
+    CMAKE_ARGS += -DJ9VM_OPT_JFR=ON
+  else # OPENJ9_ENABLE_JFR
+    CMAKE_ARGS += -DJ9VM_OPT_JFR=OFF
+  endif # OPENJ9_ENABLE_JFR
```

Comment:
This is out-of-order as well (it should go before the JITSERVER block), but there wasn't much order to begin with, so we can fix that later.